gallery_dl/extractor/wikimedia.py

```diff
@@ -17,45 +17,19 @@ class WikimediaExtractor(BaseExtractor):
     """Base class for wikimedia extractors"""
     basecategory = "wikimedia"
     filename_fmt = "{filename} ({sha1[:8]}).{extension}"
-    directory_fmt = ("{category}", "{page}")
     archive_fmt = "{sha1}"
     request_interval = (1.0, 2.0)
 
     def __init__(self, match):
         BaseExtractor.__init__(self, match)
-        path = match.group(match.lastindex)
 
         if self.category == "wikimedia":
             self.category = self.root.split(".")[-2]
         elif self.category in ("fandom", "wikigg"):
             self.category = "{}-{}".format(
```

Comment:
Should this be moved to before `BaseExtractor.__init__` is called?

Reply:
`self.category` only has a value *after* `BaseExtractor.__init__` was called, so this not really an option I think.

Reply:
Got it. It's a bit of a shame that the Fandom wikis can't be controlled individually, but I guess you could just make separate config files to achieve the same result.